src/content/events/fil-bangkok-2024.md

```diff
@@ -16,12 +16,23 @@ description: Join the Filecoin Community in Bangkok! Dive into decentralized AI
 external-link:
   url: https://lu.ma/aqyqwupe
   text: Register for Main Stage
+image:
+  src: /assets/images/fil-bangkok-2024.webp
 luma-calendar-link: https://lu.ma/filecoin?tag=fil%20bangkok%202024
 luma-events-section:
   title: FIL Bangkok 2024 Events
   embed-link: https://lu.ma/embed/calendar/cal-nlDvL4B7Ko1swF0/events?lt=light&tag=FIL%20Bangkok%202024
-image:
-  src: /assets/images/fil-bangkok-2024.webp
+schedule:
+  title: "FIL Bangkok: DePIN Meets AI Schedule"
```

Comment:
The schedule title is for the entire Schedule for all days. Knowing that, is it still supposed to be FIL Bangkok: DePIN Meets AI Schedule?

![CleanShot 2024-10-17 at 00 55 36@2x](https://github.com/user-attachments/assets/69804849-ed49-49f2-a482-6051f2e06ec8)

Reply:
Hey @mirhamasala Yes right now we will only be showing FIL Bangkok: DePIN Meets AI Schedule on Monday, November 11 because the FIL Dev Summit team wants the FDS schedule to live on [fds-5 website](https://www.fildev.io/FDS-5)